introspection-engine/connectors/sql-introspection-connector/tests/re_introspection/mod.rs

```diff
@@ -81,7 +81,7 @@ async fn re_introspecting_manually_overwritten_mapped_field_name(api: &TestApi)
     custom_assert(&result, final_dm);
     let warnings = api.re_introspect_warnings(input_dm).await;
 
-    assert_eq!(&warnings, "[{\"code\":8,\"message\":\"These fields were enriched with @map information taken from the previous Prisma schema.\",\"affected\":[{\"model\":\"User\",\"field\":\"test\"}]}]");
+    assert_eq!(&warnings, "[{\"code\":8,\"message\":\"These fields were enriched with @map information taken from the previous Prisma schema.\",\"affected\":[{\"model\":\"User\",\"field\":\"custom_test\"}]}]");
```

Comment:
In case you don't like comparing JSON like that: you can steal `assert_eq_json` from [here](https://github.com/prisma/prisma-engines/blob/6b10f7bfb5c09d707016877e7ec2e0c35f26eb67/libs/datamodel/core/tests/config/generators.rs#L152)